mtp_noms_ops/apps/security/views/check.py

```diff
@@ -61,48 +61,30 @@ def get_context_data(self, **kwargs):
         return context_data
 
 
-class AcceptCheckView(ActionCheckView):
+class AcceptOrRejectCheckView(BaseAcceptOrRejectCheckView):
     """
-    View accepting a check in pending status.
+    View accepting or rejecting a check in pending status.
     """
-    title = gettext_lazy('Accept this credit')
-    template_name = 'security/accept_check.html'
-    form_class = AcceptCheckForm
+    title = gettext_lazy('Review pending credit')
+    template_name = 'security/accept_or_reject_check.html'
+    form_class = AcceptOrRejectCheckForm
 
     def form_valid(self, form):
         if self.request.method == 'POST':
-            result = form.accept()
-            if not result:
-                return self.form_invalid(form)
-
-            messages.add_message(
-                self.request,
-                messages.INFO,
-                gettext_lazy('Credit accepted'),
-            )
-            return HttpResponseRedirect(self.list_url)
-
-        return super().form_valid(form)
-
+            result = form.accept_or_reject()
 
-class RejectCheckView(ActionCheckView):
-    """
-    View rejecting a check in pending status.
-    """
-    title = gettext_lazy('Reject this credit')
-    template_name = 'security/reject_check.html'
-    form_class = RejectCheckForm
-
-    def form_valid(self, form):
-        if self.request.method == 'POST':
-            result = form.reject()
             if not result:
                 return self.form_invalid(form)
 
+            if form.cleaned_data['fiu_action'] == 'accept':
+                ui_message = 'Credit accepted'
```

Comment:
you need the `gettext` wrapper where the string is actually defined or else the translation tools can't pick out the message